### ISSUE TITLE
'mstous' was not declared in this scope

### ISSUE THREAD

**Issue by mluis** [cyphunk/RS232enum#1]
Can't find the lib for this function. Project might need an update?

**Comment by cyphunk** [comment]
I will try to test locally soon. Which OS and Arduino version are you running the code under? Not sure why this comes up. mstous() is a function defined in _utils.ino. I can only imagine that the default behavior of the Arduino IDE to include all .ino files when compiling is somehow failing.

https://github.com/cyphunk/RS232enum/blob/master/_utils.ino#L63

```
uint32_t mstous(uint32_t ms) 
{
        return ms*1000;
}
```

**Comment by tulipszs** [comment]
Any update on this? I'm having the same issue :< Running Linux (Arch 4.20.7) and the Arduino IDE. 

- Thanks

**Comment by mluis** [comment]
I'm having the issue with OSX and Arduino IDE.

**Comment by tulipszs** [comment]
I solved the compilation error by moving the 'mstous' function definition into the main RS232enum.ino file. Although, I'm not sure why it's not functioning properly to begin with. Very odd.

**Comment by cyphunk** [comment]
iThank you for the reminder. Tested and resolved the issue:
https://github.com/cyphunk/RS232enum/commit/e187a9c32c9515db3e295f57374464844d656b66

I was able to replicate on another archlinux system. Indeed it is odd. I seem to remember running into this issue with Arduino where the abstraction it creates when including other files isn't failproof. The solution I chose was to add a declaration for the function in the main .ino.

Let me know if you run into any other issues.
